certbot/CHANGELOG.md

```diff
@@ -26,6 +26,8 @@ More details about these changes can be found on our GitHub repo.
   this concerns the plugin name, CLI flags, and keys in credential files.
   The prefixed form is still supported but is deprecated, and will be removed in a future release.
 * Added `--nginx-sleep-seconds` (default `1`) for environments where nginx takes a long time to reload.
+* Added the ability to remove email and phone contact information from an account 
+  using `update_account --register-unsafely-without-email`
 
 ### Changed
 
```

Comment:
nit: There aren't many, but there are a few projects using our `acme` module as a library. Can we add a "fixed" section describing how a contact field that is explicitly set to an empty tuple is now serialized and sent to the ACME server allowing people to remove contact information from an existing registration?